### PR TITLE
LPD-18438 Add a pause before select the workflow definition

### PR DESCRIPTION
**Pull request by seltong**
Related issues:

- [LPD-18438](https://liferay.atlassian.net/browse/LPD-18438)
- [LPD-18465](https://liferay.atlassian.net/browse/LPD-18465)

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/Workflow.macro [hunk 1]
```diff
@@ -407,8 +407,12 @@ definition {
 
 		Click(locator1 = "Icon#SIDEBAR_HEADER_ACTION_VERTICAL_ELLIPSIS_BUTTON");
 
+		Pause(value1 = 5000);
+
 		MenuItem.click(menuItem = "Edit");
 
+		Pause(value1 = 5000);
```
Comment (MarinhoFeliphe) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/Workflow.macro:414]: Hey @seltong,

use `WaitForElementPresent` or something like this:

```
while ((IsElementNotPresent(locator1 = "WorkflowAllItems#WORKFLOW_ALL_ITEMS_FILTER_BY_ASSIGNEE_VALUE")) && (maxIterations = "10")) {
				Refresh();

				WaitForSPARefresh();

				Click(locator1 = "WorkflowAllItems#WORKFLOW_ALL_ITEMS_FILTER_BY");
			}
```

we've talked to @Luizcarlosqueiroz and he said the `Pause` function is no longer used and we should avoid

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by MarinhoFeliphe** [comment]
ci:test:workflow

**Comment by seltong** [comment]
Hi @MarinhoFeliphe, the issue still failing in the CI. I will analyze what is happening and send another PR.

Thank you!

**Comment by seltong** [comment]
Resent here: https://github.com/liferay-workflow/liferay-portal/pull/1089

**Pull request closed without merging**